watertap/tools/oli/periodic_table.csv

```diff
@@ -0,0 +1,119 @@
+AtomicNumber,Element,Symbol,AtomicMass,NumberOfNeutrons,NumberOfProtons,NumberOfElectrons,Period,Group,Phase,Radioactive,Natural,Metal,Nonmetal,Metalloid,Type,AtomicRadius,Electronegativity,ionizationEnergy,Density,MeltingPoint,BoilingPoint,stableIsotopes,Discoverer,Year,SpecificHeat,NumberOfShells,NumberOfValence
```

Comment:
I wonder if we want a test that ensures that state of this csv is preserved. That is, if any developer ever comes up with a need to modify this is any way, we should have a way to control such modifications to prevent unintended consequences (i.e., any changes to this csv would have to be reflected in the corresponding test file). For example, we can have a test that checks that all the headers currently in the file match the expected list.

Reply:
I put up a simple test in the latest commit

Reply:
I slightly wounder, if its better to convert this to json/yaml. so its important as a proper python dictionary from getgo? Might be easier to manage. Just a minor suggestion.

Reply:
On other note. I wounder if we should instead rely on a library designed for handling this sort of thing like molmass https://pypi.org/project/molmass/  ... it might remove all the headaches, but does add a dependency?